addons/talo/apis/leaderboards_api.gd

```diff
@@ -22,13 +22,13 @@ func get_cached_entries_for_current_player(internal_name: String) -> Array:
 	)
 
 ## Get a list of entries for a leaderboard. The page parameter is used for pagination.
-func get_entries(internal_name: String, page: int, alias_id = -1, include_archived = false) -> Array:
+func get_entries(internal_name: String, page: int, alias_id = -1, include_archived = false) -> EntriesPage:
 	var url = "/%s/entries?page=%s"
 	var url_data = [internal_name, page]
 
 	if alias_id != -1:
 		url += "&aliasId=%s"
-		url_data.append(alias_id)
+		url_data += alias_id
```

Comment:
looks like this is reverting another earlier fix

Reply:
Thanks for warning, already fix now.